solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -96,6 +96,20 @@ public class SolrExporter {
           + ARG_NUM_THREADS_DEFAULT
           + ".";
 
+  private static final String[] ARG_USER_FLAGS = {"-u", "--user"};
```

Comment:
at least we have `--user` instead of `-user`, over on SolrCLI, we want to get to the `--` pattern!

Reply:
I moved it all to 'credentials' similar to the example PR.